external/download.sh

```diff
@@ -14,5 +14,10 @@ while true; do
     url=$3
     read line; set -- $line
     branch=$3
+
+    if [ -z "${ARGUMENTS}" ]; then
```

Comment:
This is bash syntax. Try this?
```suggestion
    if [ "$ARGUMENTS" = "" ]; then
```

Reply:
oh, sorry. My cmake habit came into action XD. yes that works, I actually wanted to write `if [ -z "$ARGUMENTS" ]; then`. Alright, I will make the changes